security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -13,6 +13,20 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-31"]
+== OpenShift File Integrity Operator 0.1.31
```

Comment:
We hit a regression with 0.1.31 on 4.6. We already landed a fix with https://github.com/openshift/file-integrity-operator/pull/291 and we're in the process of creating 0.1.32:

https://github.com/openshift/file-integrity-operator/pull/294

Reply:
Thanks for the review and feedback! I have made the appropriate changes. :)